### ISSUE TITLE
Allow rxjava-server and rxjava2-server to co-exist

### ISSUE THREAD

**Issue by kennymacleod** [alex-shpak/rx-jersey#7]
I'm using the [RxJavaInterop](https://github.com/akarnokd/RxJava2Interop) library to allow my apps to run both RxJava1 and RxJava2 (partly to allow a gradual transition, but mostly because Hystrix is Rxjava1-only). This is fine because RxJava1 and Rxjava2 are in different Java packages and can co-exist.

However, RxJersey's rxjava-server and rxjava2-server use the same Java package (and the same class names), so you can pick one but not the other. This is a real shame, since from what I can tell they should be able to co-exist otherwise.

Any chance you could use different Java packages for each artifact?

**Comment by alex-shpak** [comment]
Hi!
This sounds reasonable, I will find separate package names.

**Comment by alex-shpak** [comment]
Do you think for RxJava 1
```
net.winterly.rxjersey.rxjava1.client
net.winterly.rxjersey.rxjava1.server
```

and for RxJava2
```
net.winterly.rxjersey.client
net.winterly.rxjersey.server
```
These are good names?
Since RxJava 1 has EOL in [March 2018](https://github.com/ReactiveX/RxJava#version-1x-javadoc)

**Comment by kennymacleod** [comment]
Yep, good idea (or at least until rxjava3 comes along...)

**Comment by alex-shpak** [comment]
After some struggle I updated packages to:
`net.winterly.rxjersey.server.rxjava`
`net.winterly.rxjersey.server.rxjava2`
etc,

since there might be more implementations, for example https://projectreactor.io/

**Comment by kennymacleod** [comment]
Was this released as part of 0.10.0? If so, it doesn't look like it was pushed to maven central yet.

**Comment by alex-shpak** [comment]
My bad 😞 forgot to press Release button, should be available now
